kubernetes/troubleshooting/README.md

```diff
@@ -0,0 +1,130 @@
+# Troubleshooting
+
+## Steps
+
+The order of troubleshooting should be as follows:
+
+1. Identify the pods you're looking for - primarily `pulumi-workflow-runner-*` pods
+   - Note: The agent pool has logs too, but they mostly show it launching runners; these may still be useful
```

Comment:
FYI, the agent pool gets logs from all workflow runners as well as showing the launch info. It's a lot of data, but it can be a good central place for troubleshooting.

Reply:
kubectl logs, usually doesn't show this much for the workflow pods, is there a debug switch we need to set to get that in there?

Reply:
Hmm, it should be the default. It may be different on older versions, though. Let's sync up offline if you're still having problems seeing it.